lib/spotify_prototype_app.dart

```diff
@@ -23,21 +25,17 @@ class SpotifyPrototypeApp extends StatelessWidget {
   }
 }
 
-List<Widget> screens() => [const PlayListScreen(), const FavouriteScreen()];
-
-List<PersistentBottomNavBarItem> navBarItems() {
-  return [
-    PersistentBottomNavBarItem(
-        icon: const Icon(
-          Icons.play_arrow,
-          color: Colors.green,
-        ),
-        title: 'Playlist'),
-    PersistentBottomNavBarItem(
-        icon: const Icon(
-          Icons.favorite,
-          color: Colors.green,
-        ),
-        title: 'FavouriteList')
-  ];
-}
+List<PersistentBottomNavBarItem> navBarItems() => [
```

Comment:
never use helper methods, refactor it to widget

Reply:
I am not sure about that, can we discuss that?